packages/builder/src/components/automation/AutomationBuilder/FlowChart/TestDataModal.svelte

```diff
@@ -15,16 +15,22 @@
   let trigger = {}
   let schemaProperties = {}
 
-  // clone the trigger so we're not mutating the reference
-  $: trigger = cloneDeep(
-    $automationStore.selectedAutomation.automation.definition.trigger
-  )
+  $: trigger
```

Comment:
These shouldn't be needed as reactive variables initially - you can just define them as `let trigger` and `let schemaProperties` below.

Reply:
Removed! thanks for that